starter-code/index.html

```diff
@@ -41,13 +41,13 @@ <h2>Your pizza's price</h2>
 
       <b>$10 cheese pizza</b>
       <ul>
-        <li>$1 pepperonni</li>
-        <li>$1 mushrooms</li>
-        <li>$1 green peppers</li>
-        <li>$3 white sauce</li>
-        <li>$5 gluten-free crust</li>
+        <li class="pep-price">$1 pepperonni</li>
```

Comment:
I like how you added a class to your ingredients to make it easier to calculate the price